Resources/config/orm.xml

```diff
@@ -233,5 +233,13 @@
 
             <tag name="console.command" command="doctrine:mapping:import" />
         </service>
+
+        <service id="Doctrine\Bundle\DoctrineBundle\Controller\ProfilerController">
+            <argument type="service" id="twig" on-invalid="ignore"/>
```

Comment:
`on-invalid="ignore"` is wrong, as the class cannot be instantiated without that argument.

Reply:
And that's fine, I don't want to that controller instantiated without those arguments, as it would be useless. It cannot work without them.

Reply:
but why adding this `on-invalid="ignore"` then ?

Reply:
Because `twig` and `profiler` are registered only in certain configurations, as discovered by tests.